assets/js/modules/adsense/datastore/service.js

```diff
@@ -125,6 +126,13 @@ export const selectors = {
 		return select( STORE_NAME ).getServiceURL( { path, query } );
 	} ),
 
+	getServiceReportURL: createRegistrySelector( ( select ) => ( state, reportArgs ) => {
+		const accountID = select( STORE_NAME ).getAccountID();
```

Comment:
When the data hasn't been loaded yet, `accountID` can be `undefined`. Can you add a condition like in the other selectors above that in such a case returns `undefined`?